UploadDaemon_Test/SymbolAnalysis/SymbolCollectionResolverTest.cs

```diff
@@ -10,56 +10,113 @@ namespace UploadDaemon.SymbolAnalysis
     [TestFixture]
     class SymbolCollectionResolverTest
     {
-        private readonly GlobPatternList someAssemblyPatterns = new GlobPatternList(new List<string> { "*" }, new List<string> { });
-        private readonly string testSymbolFileName = $"{TestUtils.TestDataDirectory}\\Some.pdb";
+        private static readonly string TestDataDirectory = $"{TestUtils.TestDataDirectory}\\Tmp";
+        private static readonly string TestSymbolFilePath = $"{TestDataDirectory}\\Some.pdb";
+
+        private readonly GlobPatternList includeAllAssembliesPattern = new GlobPatternList(new List<string> { "*" }, new List<string> { });
 
         private SymbolCollectionResolver resolver;
 
         [SetUp]
         public void SetUp()
         {
             resolver = new SymbolCollectionResolver();
-            File.SetLastWriteTime($"{TestUtils.TestDataDirectory}\\Some.pdb", new DateTime(2019, 1, 1));
+
+            Directory.CreateDirectory(TestDataDirectory);
+            File.Copy($"{TestUtils.TestDataDirectory}\\Some.pdb", TestSymbolFilePath);
+            File.SetLastWriteTime(TestSymbolFilePath, new DateTime(2019, 1, 1));
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            Directory.Delete(TestDataDirectory, true);
+        }
+
+        [Test]
+        public void ConsidersSymbolFileIncludes()
+        {
+            SymbolCollection collection = resolver.ResolveFrom(TestUtils.TestDataDirectory,
+                   new GlobPatternList(new List<string> { "DoesNotExist*" }, new List<string> {}));
+
+            Assert.That(collection.IsEmpty, Is.True);
```

Comment:
Minor: Is there a `Is.Empty` for collections? will probably give you better error messages. same for the next test

Reply:
Yes, there is, but `SymbolCollection` is not a .NET collection and, therefore, the `Is.Empty` matcher doesn't work.

Reply:
makes sense :-)